app/assets/stylesheets/components/_card-grandmother.scss

```diff
@@ -48,10 +48,16 @@
   display: grid;
   grid-template-columns: 260px 260px 260px 260px;
```

Comment:
I think here you can put 1fr 1fr 1fr 1fr so it automatically divides the grid in 4 equel colons

Reply:
for me it's ok

Reply:
thanks, will change it!